### ISSUE TITLE
Error: Evaluation failed: ReferenceError: WAPI is not defined

### ISSUE THREAD

**Issue by OmarMYousef** [orkestral/venom#191]
any solve for that error ?
i've tried to delete `node_modules` and running `npm install` and nothing changed

error appears after scanning the QR code

full error:
```
Error: Evaluation failed: ReferenceError: WAPI is not defined
    at eval (eval at waitForPredicatePageFunction (:2:23), <anonymous>:5:29)
    at eval (eval at waitForPredicatePageFunction (:2:23), <anonymous>:6:27)
    at onRaf (_puppeteer_evaluation_script_:49:35)
    at pollRaf (_puppeteer_evaluation_script_:42:15)
    at waitForPredicatePageFunction (_puppeteer_evaluation_script_:7:22)
    at ExecutionContext._evaluateInternal (C:\Users\omarm\Desktop\wao\node_modules\puppeteer\lib\ExecutionContext.js:102:19)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async WaitTask.rerun (C:\Users\omarm\Desktop\wao\node_modules\puppeteer\lib\DOMWorld.js:404:23)
  -- ASYNC --
    at ExecutionContext.<anonymous> (C:\Users\omarm\Desktop\wao\node_modules\puppeteer\lib\helper.js:94:19)
    at WaitTask.rerun (C:\Users\omarm\Desktop\wao\node_modules\puppeteer\lib\DOMWorld.js:404:71)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
C:\Users\omarm\Desktop\wao\node_modules\spinnies\index.js:107
    if (!this.spinners[name]) throw Error(`No spinner initialized with name ${name}`);
                              ^

Error: No spinner initialized with name session-auth
    at Spinnies.setSpinnerProperties (C:\Users\omarm\Desktop\wao\node_modules\spinnies\index.js:107:37)
    at Spinnies.fail (C:\Users\omarm\Desktop\wao\node_modules\spinnies\index.js:69:10)
    at Timeout._onTimeout (C:\Users\omarm\Desktop\wao\node_modules\venom-bot\dist\controllers\initializer.js:177:38)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
```

**Comment by midincihuy** [comment]
I think it's solved in newest version, I used version 2.0.5,
change in package.json 
then run 
`npm update`

**Comment by OmarMYousef** [comment]
> I think it's solved in newest version, I used version 2.0.5,
> change in package.json
> then run
> `npm update`
